uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/GeneralSettings.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+require_once 'CRM/Core/Form.php';
+
+/**
+ * Form controller class
+ *
+ * @see http://wiki.civicrm.org/confluence/display/CRMDOC43/QuickForm+Reference
+ */
+class CRM_HRLeaveAndAbsences_Form_GeneralSettings extends CRM_Core_Form {
+
+  /**
+   * @var array
+   *   The filter to pass to the setting.getfields API
+   */
+  private $_settingFilter = ['group' => 'leaveandabsence'];
```

Comment:
you're already using the `private` modifier, so there's no need to prefix the variables with an underscore